<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QWidget" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="searchInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgb(0, 0, 0);
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-top-left-radius: 5px;
border-bottom-left-radius: 5px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgb(0, 0, 0);
border-top-right-radius: 5px;
border-bottom-right-radius: 5px;
border-top-left-radius: 0;
border-bottom-left-radius: 0;</string>
   </property>
   <property name="text">
    <string>검색</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="searchTabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>881</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
 background-color: rgb(255, 255, 255);
}
QHeaderView::section {
    background-color: rgba(202,205,205,0.5);
    border: 1px solid rgba(0, 0, 0, 0.5);
    padding: 3px;
    font-weight: bold;
    text-align: center;
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="customerTab">
    <attribute name="title">
     <string>Customer</string>
    </attribute>
    <widget class="QTableView" name="customerTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="textLabel">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>200</y>
       <width>331</width>
       <height>171</height>
      </rect>
     </property>
     <property name="text">
      <string>고객 정보</string>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>10</y>
       <width>331</width>
       <height>181</height>
      </rect>
     </property>
     <property name="text">
      <string>차량 번호판</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="videoTab">
    <attribute name="title">
     <string>Video</string>
    </attribute>
    <widget class="QTableView" name="videoTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>851</width>
       <height>351</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_edit">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>350</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>EDIT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>380</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_enroll">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>320</y>
     <width>88</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ENROLL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="filterButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgb(0, 0, 0);
border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-top-left-radius: 0;
border-bottom-left-radius: 0;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../images.qrc">
     <normaloff>:/images/list.png</normaloff>:/images/list.png</iconset>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../images.qrc"/>
 </resources>
 <connections/>
</ui>
